02-microarray/pathway-analysis_microarray_03_gsea.Rmd

```diff
@@ -333,15 +338,169 @@ Looks like we were able to successfully get rid of the duplicate gene identifier
 
 ## Perform gene set enrichment analysis (GSEA)
 
-_Addressed in upcoming PR_
+The goal of GSEA is to detect situations where all genes in a gene set change in a coordinated way, detecting even small statistical but coordinated changes.
+This is done by ranking genes within a gene set from most highly positive to most highly negative, weighting them according to their gene-level statistic, to calculate the enrichment score (ES), which is a pathway-level statistic [@clusterProfiler-book].
+
+### Determine our pre-ranked genes list
+
+The `GSEA()` function takes a pre-ranked and sorted named vector of statistics, where the names in the vector are gene identifiers.
+This is step 1 -- create a gene list with statistics that GSEA will rank by.
```

Comment:
I feel like this may be leftover from training material where we say more about steps? Or maybe the training material doesn't say anything about steps after 1 #itwasapilot.